app/controllers/contributors_controller.rb

```diff
@@ -24,6 +24,7 @@ def index
     @state = state_params
     @tag_list = tag_list_params
 
+    #TODO: scope by organization?
```

Comment:
Implemented in https://github.com/tactilenews/100eyes/pull/1898/files#diff-1bd0a0fa1f76adfb12b7cc47fc7ab252b034c736ef7006a51b7353c10c581931R26-R28, but it's probably not easy to cherry pick the changes.

Still we should go through that PR since I caught a lot of these there.

Reply:
thnx. done with: https://github.com/tactilenews/100eyes/pull/1987